README.md

```diff
@@ -1,14 +1,7 @@
 <div style="text-align: center;">
 <img src="https://cdn.discordapp.com/attachments/456120666874183680/575235193384861716/couronne.png" style="border-radius: 50%; width: 300px" alt="Couronne DraftBot">
 
-DraftBot is a bot developed in Node.js by [@BastLast](https://github.com/BastLast) based on a text adventure game. The
-story takes place in a medieval world, and the player embodies an adventurer who wants to win a competition launched by
-the king, to win the princess' hand. To achieve this, he must travel and earn points by going through a lot of danger,
-including thieves, wilderness, and scary children. Every few hours, the player can get a random event by using
-the `/report` command: they interact with a multiple choice system, using reactions below the report. Each choice has
-multiple issues, which could hurt the character, heal them, or even give them some stuff. During the journey, they earn
-money, stuff, and points, and can use them to buy potions, stuff or effects in the shop, fight other players, and ascend
-the leaderboard.
+DraftBot is a text adventure game. The story takes place in a medieval world, and the player embodies an adventurer who wants to win a competition launched by the king, to win the princess' hand. To achieve this, he must travel and earn points by going through a lot of danger, including thieves, wilderness, and scary children. Every few hours, the player can follow their adventure through "reports" and  interact with a multiple choice system, using reactions below the report. Each choice has multiple issues, which could hurt the character, heal them, or even give them some stuff. During the journey, they earn money, stuff, and points, and can use them to buy potions, stuff or effects in the shop, fight other players, and ascend the leaderboard.
```

Comment:
Y'a un double espacement (involontaire) à "and interact with"... et beaucoup de répétitions de "stuff" sur les dernières phrases, visiblement.